tests/queries/0_stateless/01526_client_start_and_exit.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
-# Tags: no-fasttest
+# Tags: no-fasttest, no-parallel
```

Comment:
I would be ok if we remove or disable this test. It's doing a crazy things to the server to check a basic/corner case bug in the client. It's not super useful.

WDYT?

Reply:
I don't have strong opinion about that tests. I could only say that makeing 10K tables to check something in the client is not optimal. I do not understand why is there 10K instead 10?

Reply:
I would just remove it. We are trying to get rid of slow no-parallel tests too, so I would not bother.

Reply:
Then lets remove it in that PR with all that context. I will do that.